### ISSUE TITLE
Input select with empty value in last option

### ISSUE THREAD

**Issue by polianomartini** [sweetalert2/sweetalert2#1757]
I'm creating an input of type select informing the options:

```javascript
Swal.fire({
  title: 'Select example',
  input: 'select',
  inputOptions: {
    '': '',
    '1': 'a',
    '2': 'b',
  },
  inputValue: '1',
  showCancelButton: true
});
```

But the empty value option is rendered at the end of the list and not at the beginning as added in the inputOptions property.

![sweetalert2_select](https://user-images.githubusercontent.com/13044849/66070155-2e9c6f00-e527-11e9-8c03-befe135366ef.png)

HTML:
```html
<select class="swal2-select" style="display: flex;">
    <option value="1">a</option>
    <option value="2">b</option>
    <option value=""></option>
</select>
```

**The input may come with an initial value and the user may not want any value and select an empty option.**

**Comment by limonte** [comment]
Duplicate of #812 read it throught to figure out the reason.

TLDR: use `Map`:

```js
Swal.fire({
  title: 'Select example',
  input: 'select',
  inputOptions: new Map([
    ['', ''],
    [1, 'a'],
    [2, 'b']
  ]),
  inputValue: '1',
  showCancelButton: true
});
```

**Comment by polianomartini** [comment]
> ```js
> Swal.fire({
>   title: 'Select example',
>   input: 'select',
>   inputOptions: new Map([
>     ['', ''],
>     [1, 'a'],
>     [2, 'b']
>   ]),
>   inputValue: '1',
>   showCancelButton: true
> });
> ```

I ran this code but select didn't load the options, it was empty.

```html
<select class="swal2-select" style="display: flex;"></select>
```

**Comment by limonte** [comment]
Just tested it out on https://sweetalert2.github.io/ works as expected.

**Comment by polianomartini** [comment]
Excuse me! It really is correct. Thanks!
